### ISSUE TITLE
Wiki and doc in english

### ISSUE THREAD

**Issue by Skallwar** [rcore-os/zCore#172]
Hi,

I want to contribute to zCore but I must say that I'm having an hard time due to the fact that most of the doc is in Chinese. Is there a English version of the wiki and documentation somewhere ?

**Comment by yunwei37** [comment]
May be you can try to read the cargo doc inside the code? all the cargo docs here are in English.

**Comment by Skallwar** [comment]
Yes but the wiki is not. For example the [contribution page](https://github.com/rcore-os/zCore/wiki/Contribution) is the first page new contributors will try to read.

And yeah the cargo doc is great but only covers the kernel objects, which might not be enough to start contributing
